Doc/cmus.txt

```diff
@@ -1666,6 +1673,45 @@ USER
 USERNAME
 	Fallback for *USER*.
 
+@h2 Library Environment Variables
+
+cmus supports using environment variables to allow the library, cache, and
+playlists to be relocated or synced. The environment variables to use are
+specified using the `pl_env_vars` configuration option, which takes effect when
+exiting cmus.
+
+A good starting point is `:set pl_env_vars=HOME`. This will allow the `.cmus`
+directory to be shared between devices or users. Additional environment
+variables can be appended (delimited with commas) to `pl_env_vars` to redirect
+specific paths. The environment variables must contain absolute paths to be
+used. The last matching directory prefix will be used.
+
+If a file does not exist, it will be treated the same way as missing files
+usually are. If the file is in the cache, it will be preserved, and an error
+will be displayed upon playback. If the file is not in the cache, it will be
+removed from playlists and the library. There is one special exception: if a
+file is missing due to the environment variable being invalid/empty/unset, but
+it is still in the cache, it will be preserved as-is.
```

Comment:
Usually, cmus will remove files if they aren't in the filesystem. I've added an exception to the default behaviour to keep the file if the entry with the env var placeholder is still in the cache. I'm not sure if there's a less technical way to explain this.

Reply:
I'm still not sure, but I think it could be that the first half of your description is referring to cmus startup and the last part is referring to `:update`? At least the normal behaviour is different for both: on startup, missing files are kept if they're in the cache, while for `:update` missing files are always removed. It would make sense if you've added the exception to `:update`. But I suspect I'm still not getting it. ;)

Reply:
Updated.